CMakeLists.txt

```diff
@@ -539,7 +539,9 @@ endif()
 
 option(AVIF_BUILD_TESTS "Build avif tests." OFF)
 if(AVIF_BUILD_TESTS)
-    # Reference tests for CTest to work.
+    # With testing enabled, all targets referenced by add_test() can be run with
+    # CMake's ctest command line tool from the build folder: ctest [options]
+    # Options include filtering tests with a regexp -R, multithreading -j etc.
```

Comment:
Thank you! I didn't realize that we can just say "ctest" without any options. Given that, we can omit everything after "from the build folder" if you prefer a concise comment.